scripts/helm/publish-chart-yaml.sh

```diff
@@ -409,7 +416,35 @@ else
     if [ -n "$APP_TAG" ]; then
       die "Cannot specify --update-release and --app-tag together"
     fi
-    APP_TAG=$(semver bump "patch" "$UPDATE_REL")
+    if [ "$(semver get build "$UPDATE_REL")" != "" ]; then
+      die "Build not supported"
+    fi
+    if [ -n "$BRANCH" ]; then
+      if ! [[ "$BRANCH" =~ ^release/[0-9]+.[0-9]+$ ]]; then
```

Comment:
```suggestion
      if ! [[ "$BRANCH" =~ ^(release/[0-9]+.[0-9]+)$ ]]; then
```

Reply:
We're not capturing groups here, so there's no need for that? Or is there another reason I'm unaware of?